src/include/catalog/pg_proc.dat

```diff
@@ -11085,25 +11085,25 @@
   proname => 'gp_percentile_cont_float8_transition',
   proisstrict => 'f',
   prorettype => 'float8',
-  proargtypes => 'float8 float8 int8 int8',
+  proargtypes => 'float8 float8 float8 int8 int8',
```

Comment:
@RekGRpth why didn't you change also [pg_aggregate.dat](https://github.com/arenadata/gpdb/blob/477b04a305825dd75d471b981a06ef611bbe2b09/src/include/catalog/pg_aggregate.dat#L677). What are the problems can be if will be found incorrect aggfnoid for PG_AGGREGATE?

Reply:
I don't get it, please clarify your idea. The `gp_percentile_cont` function has FOUR arguments, while the `gp_percentile_cont` function has FIVE arguments! What do you think I should have changed in the `pg_aggregate.dat` file?

Reply:
Oh yes, I confused everything, now I realized that the link works as it should
```sql
-[ RECORD 156 ]--+------------------------------------------
aggfnoid         | pg_catalog.gp_percentile_cont
aggkind          | n
aggnumdirectargs | 0
aggtransfn       | gp_percentile_cont_float8_transition
aggfinalfn       | gp_percentile_final
aggcombinefn     | -
...
```